src/Nest/CommonOptions/Fuzziness/FuzzinessJsonConverter.cs

```diff
@@ -1,19 +1,28 @@
 ﻿using System;
+using System.Text.RegularExpressions;
 using Newtonsoft.Json;
 
 namespace Nest
 {
 	internal class FuzzinessJsonConverter : JsonConverter
 	{
+		private static readonly Regex AutoLengthRegex = new Regex(@"^AUTO:(?<low>\+?\d+),(?<high>\+?\d+)$", RegexOptions.Compiled | RegexOptions.IgnoreCase);
```

Comment:
I suspect looking for the index of `:` and `,` will be faster than a regex here.

Reply:
Changed. it is now more simple, but may throw an exception when source string is malformed (for example when ```,``` is before ```;```

Reply:
I think this is acceptable here, since Elasticsearch constrains the format of the response:

https://github.com/elastic/elasticsearch/blob/3b8c2e4d8e221677324753cd3265cc111d0faa95/server/src/main/java/org/elasticsearch/common/unit/Fuzziness.java#L69-L77